odoo-shippable/scripts/library.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+git_clone_copy(){
```

Comment:
FYI we need the `.git` folder for `odoo/odoo` case, because many times we use a dummy PR to test other version of `odoo` [E.g.](https://github.com/Vauxoo/yoytec/pull/958/files) and we don't like download the full repository each `t2d build`.

We need save time in each `t2d build` adding a cache repository for `odoo` in this image.
But we need the download process in MQT to compatibility with `travis` (don't use odoo cached).